Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -297,6 +298,51 @@ public IEnumerator PressButtonFast([ValueSource(nameof(PressableButtonsTestPrefa
             yield return null;
         }
 
+        /// <summary>
+        /// Tests if button pressed event is triggered when a second button is overlapping right behind the first button
+        /// </summary>
+        [UnityTest]
+        public IEnumerator PressButtonWhenSecondButtonIsNearby([ValueSource(nameof(PressableButtonsTestPrefabFilenames))] string prefabFilename)
+        {
+            GameObject testButton1 = InstantiateDefaultPressableButton(prefabFilename);
+            GameObject testButton2 = InstantiateDefaultPressableButton(prefabFilename);
+
+            // Move the camera to origin looking at +z to more easily see the button.
+            TestUtilities.PlayspaceToOriginLookingForward();
+
+            PressableButton buttonComponent = testButton1.GetComponent<PressableButton>();
+            Assert.IsNotNull(buttonComponent);
+            Assert.IsTrue(buttonComponent.EnforceFrontPush, "Button default behavior should have enforce front push enabled");
+
+            // Positioning the start push plane of the second button just before first button max push plane, creating a small overlap
+            float distance = Mathf.Abs(buttonComponent.MaxPushDistance) + Mathf.Abs(buttonComponent.StartPushDistance);
+            distance = buttonComponent.DistanceSpaceMode == SpaceMode.Local ? distance * buttonComponent.LocalToWorldScale : distance;
+            testButton2.transform.position += Vector3.forward * distance;
+            
+            bool buttonPressed = false;
+            buttonComponent.ButtonPressed.AddListener(() =>
+            {
+                buttonPressed = true;
+            });
+
+            // Move the hand so the pointer passes through the two buttons 
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            float handReach = 0.1f;
+            int numSteps = (int)Mathf.Ceil(handReach / (distance * 0.5f)); // Maximum hand speed in order to trigger touch started in the first button before the second button becomes the closest touchable to pointer
+            Vector3 p1 = new Vector3(0, 0, -handReach/2);
+            Vector3 p2 = new Vector3(0, 0, handReach/2);
+
+            yield return PlayModeTestUtilities.ShowHand(Handedness.Right, inputSimulationService);
```

Comment:
can you use TestHand instead of this util method? would be easier to use and easier to read

Reply:
I feel like I've seen a lot of comments like this recently about this API. Do we want to deprecate it and migrate everything over to `TestHand`? Are there still cases where this API is "better"?

Reply:
`TestHand` is just a wrapper on  PlayModeTestUtilities.ShowHand and other PlayModeTestUtilities fucntions so they can't be deprecated

Reply:
Changed all calls to PlayModeTestUtilities.ShowHand, move or hide found in the file to TestHand

Reply:
@keveleigh - I think this is more a copy paste thing - we never unified the tests to stick with the one or the other - it's ok to use either of them but imho the wrapper is just easier to read / nicer to use - I'll open a ticket to go over the tests and replace the utility calls with the test hand